tests/test_receivers.py

```diff
@@ -65,3 +67,26 @@ def test_build_cache_with_no_collections(app):
 
         # now the collection 'mycoll' should appear in `record0` of A!
         assert record0['_collections'] == ['mycoll']
+
+
+def test_cache_can_be_imported_from_a_config_variable(app):
+    InvenioCollections(app)
+    app.config['COLLECTIONS_CACHE'] = 'werkzeug.contrib.cache:SimpleCache'
+
+    assert current_collections.cache is not None
+
+
+def test_cache_can_be_passed_from_a_config_variable(app):
+    InvenioCollections(app)
+    app.config['COLLECTIONS_CACHE'] = SimpleCache()
+
+    assert current_collections.cache is not None
```

Comment:
Doing something like this would be enough?

```
simple_cache = SimpleCache()
app.config['COLLECTIONS_CACHE'] = simple_cache

assert current_collections.cache is simple_cache
```